openlibrary/core/admin.py

```diff
@@ -154,20 +154,65 @@ def _get_count_docs(ndays):
     return [d for d in docs if d]
 
 
-def get_stats(ndays=30):
+def get_stats(ndays=30, dev_prod=False):
```

Comment:
```suggestion
def get_stats(ndays=30, use_mock_data=False):
```
`dev_prod` doesn't really convey what this boolean is used for.